anonymize/src/main/java/org/datahem/processor/anonymize/AnonymizeStreamPipeline.java

```diff
@@ -335,12 +313,11 @@ public static void main(String[] args) throws IOException {
 					new SerializableFunction<String, String>() {
 						@Override
 						public String apply(String tableSpec) {
-							return tableSpec.replaceAll("[^A-Za-z0-9.]", "");
+							return tableSpec.replaceAll("[^A-Za-z0-9._]", "");
 						}
```

Comment:
Sorry for being slow, but what do these lines actually do?

Reply:
It looks like it clears some fields in rows that are to be written to recovery table. Doesn't that defeat the purpose of the recovery table?

Reply:
it just removes characters that are not allowed in a bigquery tablename, it has nothing to do with the actual data. It's just a safety measure if user provides a table name that isn't valid.

Reply:
Ok, fine then!